File: Data Structures & Algorithms/duplicate-integer/submission-0.js
class Solution {
    /**
     * @param {number[]} nums
     * @return {boolean}
     */
    hasDuplicate(nums) {
        let duplicates = [];
        if (nums && nums.length > 0) {
          for (let i = 0; i < nums.length; i++) {
            for (let j = i + 1; j < nums.length; j++) {
              if (nums[i] == nums[j] && !duplicates.includes(nums[i])) {
                return true;
              }
            }
          }
        }
        return false;
    }
}

File: Data Structures & Algorithms/longest-consecutive-sequence/submission-0.cs
public class Solution {
    public int LongestConsecutive(int[] nums) {
        var set = new HashSet<int>(nums);

        int count = 0;
        for (int i = 0; i < nums.Length; i++)
        {
            // start tracking sequence
            if (!set.Contains(nums[i] - 1))
            {
                int length = 1;
                // continuation of number
                while (set.Contains(nums[i] + length))
                {
                    length++;
                }
                count = Math.Max(count, length);
            }
        }
        
        return count;
    }
}
